backend/inspirehep/migrator/tasks.py

```diff
@@ -344,11 +344,16 @@ def migrate_record_from_mirror(
     logger = LOGGER.bind(recid=prod_record.recid)
     try:
         json_record = marcxml2record(prod_record.marcxml)
+    except NotSupportedError as exc:
+        logger.warning(str(exc), recid=prod_record.recid)
```

Comment:
Do we have guidelines about logging priority somewhere? to me this is an `info` rather than a `warning`.

Reply:
To mee too. Apparently, during our refinement people agreed that this is a warning, thus I am sticking to our requirements.

Reply:
If you are also for `info` than we can reevaluate it.